redwood-treehouse-host/src/androidInstrumentedTest/kotlin/app/cash/redwood/treehouse/TreehouseWidgetViewTest.kt

```diff
@@ -113,6 +117,26 @@ class TreehouseWidgetViewTest {
     }
   }
 
+  @Test fun hostConfigurationEmitsSafeAreaInsetsChanges() = runTest {
+    val layout = TreehouseWidgetView(context, throwingWidgetSystem)
+    layout.hostConfiguration.test {
+      assertEquals(HostConfiguration(safeAreaInsets = Margin.Zero), awaitItem())
+      val insets = Insets.of(10, 20, 30, 40)
+      val windowInsets = WindowInsetsCompat.Builder()
+        .setInsets(WindowInsetsCompat.Type.systemBars(), insets)
```

Comment:
Add a test for IME insets (IME visible) too?

Reply:
@chrisbanes Hmm doesn't seem like there's an easy way to test IME inset changes (`dispatchApplyWindowInsets` strips those insets), but let me know if I'm missing something!

Reply:
Ahhh, this is an instrumented test. Yeah this is going to be difficult as you'll have to manually show the IME (which is a nightmare on emulators). Let's skip 👍